src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -924,6 +932,44 @@ bool OutputStateMachineEngine::_ParseHyperlink(const std::wstring_view string,
     return false;
 }
 
+// Routine Description:
+// - OSC 9 ; 4 ; state ; progress ST
+//       state: the state to set the taskbar progress indicator to
+//       progress: the progress value to set the taskbar progress indicator with
+// - Parses out the 'state' and 'progress' parameters from the OSC string so
+//   that we can use them to set the taskbar progress indicator

```

Comment:
Maybe add the link to the ConEmu documentation to explain what the expected behaviour is.